src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -242,46 +244,50 @@ <H2>Address Cache</H2>
 <a id="Unixdomain"></a>
 <H2>Unix domain sockets</H2>
-<P>There are a number of system (and networking) properties that affect the behavior of
-channels to <i>Unix domain</i> server sockets when binding <i>automatically</i>.
 <p>
-Automatic binding of a server socket occurs when {@link 
-java.nio.channels.ServerSocketChannel#bind(SocketAddress,int) ServerSocketChannel.bind}
-is called with a {@code null} address parameter. In this case, the system
-is requested to choose a unique path for the socket in some predefined system temporary
-directory.
+Calling {@link java.nio.channels.ServerSocketChannel#bind(SocketAddress,int) ServerSocketChannel.bind}
+with a {@code null} address parameter will bind to an <i>automatically assigned</i> socket address.
+For Unix domain sockets, this means a unique path in some predefined system temporary directory.
+<P>There are a number of system (and networking) properties that affect this behavior.
 <p>
-Bearing in mind that Unix domain socket addresses are limited in length to
-approximately 100 bytes, it is important to ensure that the temporary directory's name
-together with the filename used for the socket (currently a name of the form
+Bearing in mind that Unix domain socket addresses are limited in length to approximately 100 
+bytes (depending on the platform), it is important to ensure that the temporary directory's name
+together with the filename used for the socket (currently a name similar to
 {@code socket_1679697142}) does not exceed this limit. The following properties
 can be used to control the selection of this directory:
 <p>
 <ul>
-	<LI><P><B>{@systemProperty jdk.net.unixdomain.tmpdir}</b> This can be set as 
+	<li><p><b>{@systemProperty jdk.net.unixdomain.tmpdir}</b> This can be set as 
 	a networking property in {@code conf/net.properties} If set, this specifies the 
 	directory to use for automatically bound server socket addresses. On some platforms,
 	(eg some Unix systems) this will have a predefined default value. On others,
 	(eg Windows) there is no default value. Either way, it is always possible
 	to override the networking property with a system property of the same name
 	set on the command line. If neither of the networking nor system property
-	are set, then some systems may check a commonly used environment variable as temporary
-	directory.
-	<LI><P><B>{@systemProperty java.io.tmpdir}</b> If the previous step fails to locate
+	are set, then some systems (eg Windows)  may check a commonly used environment 
+	variable as temporary directory.
+	<li><p><b>{@systemProperty java.io.tmpdir}</b> If the previous step fails to locate
 	a directory to use, then the directory identified by the system property
 	{@code java.io.tmpdir} is used.
 </ul>
 More information about the platform specific behavior can be seen in the 
 {@code conf/net.properties} configuration file.
-
+<p>
+<i>Implicit</i> binding of {@link java.nio.channels.SocketChannel SocketChannel}s
```

Comment:
Nit: could this be changed to:
```
<i>Implicit</i> binding of  a {@link java.nio.channels.SocketChannel SocketChannel}
```

Reply:
Done. Would you mind adding yourself as reviewer of the CSR?